packages/jest-diff/src/__tests__/diff.test.ts

```diff
@@ -5,10 +5,9 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import ansiRegex = require('ansi-regex');
-import * as style from 'ansi-styles';
 import chalk from 'chalk';
 import stripAnsi from 'strip-ansi';
+import convertAnsi1 from '../../../pretty-format/src/plugins/ConvertAnsi1';
```

Comment:
Not a fan of these import paths :/ If we can come up with a good name for it, maybe it's fine to put it in test utils after all? It is only for our tests anyway so...

Reply:
Yes, indeed. Thanks for your patience when I didn’t get your super suggestion about `test-utils` the first time ❤️

Reply:
Do I need to add `{"path": "../test-utils"}` to `tsconfig.json` file in packages that use it?

Reply:
Yes